bindings.cpp

```diff
@@ -2881,7 +2895,7 @@ void DeRestPluginPrivate::checkOldSensorGroups(Sensor *sensor)
                     i->removeDeviceMembership(sensor->id());
                 }
 
-                if (i->state() == Group::StateNormal && !i->hasDeviceMembers())
+                if (i->address() != 0 && i->state() == Group::StateNormal && !i->hasDeviceMembers())
```

Comment:
There's several things going wrong with `/groups/0`.  Events are issues on `/groups/65520`  instead of on `/groups/0`.  A GET of `/groups/65520` returns the same as a GET of `/groups/0`.  The entire code handling this should be reviewed.  I think we might need to decouple the REST resource ID from the group address for all groups, similar to the Hue bridge.